packages/fluentui/react-icons-northstar/src/components/SpeakerLouderIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const SpeakerLouderIcon = createSvgIcon({
```

Comment:
Should be this just `Speaker` icon?

Reply:
Exists in FluentUI/Stardust as VolumeUpIcon